packages/app-store/applecalendar/api/add.ts

```diff
@@ -60,6 +60,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         id: 0,
         ...data,
         user: { email: user.email },
+        delegationCredentialId: null,
```

Comment:
added this small change in many files as there could be `delegationCredentialId` in credential table now. I didn't want to keep the prop as optional in TS. This is to ensure that if we forget to select `delegationCredentialId` in prisma query we don't endup breaking the functionality.

Reply:
Can you provide an example? I think we should treat it as optional tho. I don't want to introduce unnecessary prop drilling